data/releases/schedule.yaml

```diff
@@ -4,10 +4,15 @@ schedules:
   maintenanceModeStartDate: 2023-12-28
   endOfLifeDate: 2024-02-28
   next:
-    release: 1.26.2
-    cherryPickDeadline: 2023-02-10
-    targetDate: 2023-02-15
+    release: 1.26.3
+    cherryPickDeadline: 2023-03-10
+    targetDate: 2023-03-15
   previousPatches:
+    - release: 1.26.2
+      cherryPickDeadline: 2023-02-10
+      targetDate: 2023-02-15
```

Comment:
Should this that be the original target date or when we actually did the release (February 28)?

Reply:
When we actually released.